app/mixins/event-wizard.js

```diff
@@ -3,6 +3,7 @@ import MutableArray from '@ember/array/mutable';
 import RSVP from 'rsvp';
 import { v1 } from 'ember-uuid';
 import CustomFormMixin from 'open-event-frontend/mixins/custom-form';
+import { allSettled } from 'rsvp';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['rsvp' import is duplicated.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5583663)